<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame_toolbuttons</class>
 <widget class="QWidget" name="Frame_toolbuttons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>56</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="toolB_play">
     <property name="minimumSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Play/pause animation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/icons/play.png</normaloff>resource/icons/play.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolB_stop">
     <property name="minimumSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Stop animation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/icons/stop.png</normaloff>resource/icons/stop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolB_prev">
     <property name="minimumSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Previous animation frame&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/icons/backward.png</normaloff>resource/icons/backward.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolB_next">
     <property name="minimumSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Next animation frame&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/icons/forward.png</normaloff>resource/icons/forward.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
